### PR TITLE
Felinid rework 2 electric boogaloo

### PR DESCRIPTION
**Pull request by jbox144**
## About The Pull Request

We back

I fixed it up and removed all the downsides this PR brought. Now felinids simply get the following changes:

- New claws mutation, which felinids are predisposed to. Grants 3x unarmed melee damage, bringing punches from 7 damage to 21 damage, and granting the "clawed" effects similar to moths. Incompatible with hulk, for obvious reasons.
- New Nya mutation, a deadly mixture of genetic material that nullifies various parts of the brain. Some say it instantly makes everyone around you go into a frenzying rage.
- Felinid tails now allow felinids to instantly get up from lying down, instead of the 10 tick delay normally applied
- Tails now grant invulnerability to knockdown during shuttle flights

- Moths now get slight night vision, same as apids do. Why do apids get all the cool stuff?

## Why It's Good For The Game

Gives felinids a reason to exist.

## Changelog
:cl:
add: Added "claws" mutation, which felinids are predisposed to. Grants 3x unarmed melee damage. Incompatible with hulk.
add: Added "Nya" mutation, a deadly mutation that inhibits various parts of the brain.
tweak: Cat tails now allow you to get up instantly from lying down, instead of the normal 10 tick delay
tweak: Tails provide immunity to shuttle knockdown
balance: Moths now have night vision on par with Apids. Why do apids get all the cool stuff?
/:cl:

### CODE REVIEW

File: beestation.dme [hunk 1]
```diff
@@ -2025,9 +2025,9 @@
 #include "code\modules\language\mushroom.dm"
 #include "code\modules\language\narsian.dm"
 #include "code\modules\language\piratespeak.dm"
-#include "code\modules\language\shadowtongue.dm"
 #include "code\modules\language\ratvarian.dm"
 #include "code\modules\language\rlyehian.dm"
+#include "code\modules\language\shadowtongue.dm"
```
Comment (TheFakeElon) [inline beestation.dme:2030]: reee

File: code/modules/mob/living/carbon/human/species_types/felinid.dm [hunk 2]
```diff
@@ -89,7 +89,7 @@
 			NT.Insert(H, drop_if_replaced = FALSE)
 		else
 			tail.Remove(H)
-			
+
```
Comment (TheFakeElon) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm:92]: reeee

**Comment by ZombiesVsPlants** [comment]
stealing felind tails new meta?

**Comment by Doosteroni** [comment]
What does the "Nya" mutation actually do? You gotta specify. Also In what world is giving felinids MORE UNARMED DAMAGE THEN THE CHAPLAINS NULLROD!

**Comment by Doosteroni** [comment]
From what I've seen "Nya" just replaces your text. In what world, WHAT WORLD is this balanced

**Comment by ErwinSnowden** [comment]
Awful changes because A. 20 damage melee is better than (most) martial arts. B. Anti-knockdown sounds simply awful just because you picked a race roundstart. C. this offers nothing in the negative spectrum

**Comment by TheFakeElon** [comment]
> Anti-knockdown sounds simply awful just because you picked a race roundstart

it's quick get up from resting, not anti knockdown.

> this offers nothing in the negative spectrum

felinids already have negatives. in fact, they only have negatives

> 20 damage melee is better than (most) martial arts

I agree here, this should be reduced to like, 12

**Comment by jbox144** [comment]
> 20 damage melee is better than (most) martial arts
@TheFakeElon @ErwinSnowden friendly reminder that hulk does basically the same thing... and grants the ability to smash down walls.

**Comment by Doosteroni** [comment]
@jbox144 I want to make something clear. Being able to break down airlocks and kill people absurdly quickly for picking a race is completely busted. I also want to remind everyone that Felinids were implemented as a **Joke** race. They were never intended to be "good". Another thing, their downsides last I checked are extremely minor. The laser pointer is literally a funny reference that never comes into play, and if I am not mistaken they have flashbang vulnerability similar to moths with welding except with moths they are completely fucked and cannot wear sunglasses to block flashes while flashbangs are rarely used by sec.

**Comment by Doosteroni** [comment]
You don't get hulk for clicking a fucking button roundstart

**Comment by jbox144** [comment]
> 
> 
> You don't get hulk for clicking a fucking button roundstart

You don't get hulk roundstart?

This is a genetic trait, you aquire it by doing gene research on a felinid.

**Comment by TheFakeElon** [comment]
> You don't get hulk for clicking a fucking button roundstart

I almost had a stroke reading that

**Comment by amidoingitright** [comment]
Somehow I suspect if these kinds of changes where on another race you wouldn't care about it either

**Comment by Doosteroni** [comment]
Oh my fucking god I have brain damage. I read that as it was a trait. I’m a retard, disregard my idiocy

**Comment by totallynotkatfood** [comment]
**not again**

**Comment by jbox144** [comment]
> **not again**

yes again

**Comment by TheFakeElon** [comment]
looks good

**Review by TheFakeElon — APPROVED** [review]
just fix line endings

**Comment by jbox144** [comment]
Wasn't sure what to do with existing files which didn't have their line endings changed, so I left them as-is

**Comment by Doosteroni** [comment]
Oh god please don’t merge this please don’t

**Comment by TheFakeElon** [comment]
remove the beestation DME change too

**Pull request merged**